src/opencl_gpg_fmt_plug.c

```diff
@@ -416,4 +416,4 @@ struct fmt_main fmt_opencl_gpg = {
 
 #endif /* plugin stanza */
 
-#endif /* HAVE_OPENCL */
+#endif /* HAVE_OPENCL && HAVE_LIBCRYPTO */
```

Comment:
You don't seem to be editing these comments like this in other files. No reason to edit just this one.

Reply:
The CPU formats have the bottom one as just `/* OpenSSL */`. Perhaps I should change them all to `/* HAVE_OPENSSL */` with a one-liner.